tests/test_interface.rb

```diff
@@ -958,6 +959,44 @@ def test_interface_ipv4_arp_timeout
     assert_raises(RuntimeError) { nonvlanint.ipv4_arp_timeout = 300 }
   end
 
+  def test_interface_fabric_forwarding_anycast_gateway
+    # Setup
+    config('no interface vlan11')
+    int = Interface.new('vlan11')
+    config('feature fabric forwarding')
+    foo = VxlanGlobal.new
+    foo.anycast_gateway_mac = '1223.3445.5668'
```

Comment:
Explicitly configuring 'feature fabric forwarding' isn't needed for VxlanGlobal/OverlayGlobal - unless it's needed for the interface property, please remove it. These three lines should just become:
```
OverlayGlobal.new.anycast_gateway_mac = '1223.3445.5668'
```